src/transformers/ava.js

```diff
@@ -73,107 +73,95 @@ export default function avaToJest(fileInfo, api, options) {
         () => detectUnsupportedNaming(fileInfo, j, ast, testFunctionName),
 
         function updateAssertions() {
```

Comment:
This looks weird. Do we have unformatted code in master??

Reply:
Seems like the first two commits should be squashed. The first does some formatting that is reverted in the second.

Reply:
I think `eslint --fix` is maybe changing code that is conflicting with `prettier`'s style?

Reply:
If you `rm -rf node_modules` to start from scratch and run `yarn`, I get all sorts of formatting lint complaints:

```bash
$ rm -rf node_modules
$ yarn
yarn install v1.5.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
[4/5] Linking dependencies...
[5/5] Building fresh packages...
$ npm run clean && npm run check:bail && npm run build

> jest-codemods@0.15.0 clean /Users/cameronhunter/workspace/github/cameronhunter/jest-codemods
> rm -rf lib coverage npm-debug.log dist


> jest-codemods@0.15.0 check:bail /Users/cameronhunter/workspace/github/cameronhunter/jest-codemods
> eslint src && npm run test:cov


/Users/cameronhunter/workspace/github/cameronhunter/jest-codemods/src/transformers/ava.js
   76:16  error  Insert `⏎················`                                                                                                                                                                                                                                                          prettier/prettier
   77:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   78:21  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   79:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   80:17  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   81:13  error  Replace `})` with `····})⏎················`                                                                                                                                                                                                                                         prettier/prettier
   82:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   83:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   84:17  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   85:17  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   86:1   error  Replace `····················logWarning(`"t.${oldPropertyName}"·is·currently·not·supported`,·p` with `························logWarning(⏎····························`"t.${oldPropertyName}"·is·currently·not·supported`,⏎····························p⏎························`  prettier/prettier
   87:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   88:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   90:17  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   91:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   92:1   error  Replace `····················` with `························`                                                                                                                                                                                                                      prettier/prettier
   93:25  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   94:21  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   95:1   error  Replace `················` with `····················`                                                                                                                                                                                                                              prettier/prettier
   96:21  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   97:1   error  Replace `························` with `····························`                                                                                                                                                                                                              prettier/prettier
   98:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
   99:21  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  100:1   error  Replace `····················` with `························`                                                                                                                                                                                                                      prettier/prettier
  101:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  102:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  103:25  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  104:29  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  105:1   error  Replace `································oldPropertyName·===·'throws'·?·'toThrow'` with `····································oldPropertyName·===·'throws'⏎········································?·'toThrow'⏎·······································`                              prettier/prettier
  106:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  107:29  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  108:1   error  Replace `························` with `····························`                                                                                                                                                                                                              prettier/prettier
  109:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  110:1   error  Replace `························` with `····························`                                                                                                                                                                                                              prettier/prettier
  111:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  112:33  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  113:1   error  Replace `····································` with `········································`                                                                                                                                                                                      prettier/prettier
  114:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  115:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  116:29  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  117:25  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  118:1   error  Replace `····················` with `························`                                                                                                                                                                                                                      prettier/prettier
  119:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  120:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  121:25  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  123:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  124:25  error  Replace `logWarning(`"t.${oldPropertyName}"·should·have·2·arguments`,·p` with `····logWarning(⏎································`"t.${oldPropertyName}"·should·have·2·arguments`,⏎································p⏎····························`                                    prettier/prettier
  125:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  126:1   error  Replace `····················` with `························`                                                                                                                                                                                                                      prettier/prettier
  128:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  129:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  131:21  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  132:25  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  133:1   error  Replace `························` with `····························`                                                                                                                                                                                                              prettier/prettier
  134:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  135:25  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  136:1   error  Replace `····························` with `································`                                                                                                                                                                                                      prettier/prettier
  137:25  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  138:1   error  Replace `····················` with `························`                                                                                                                                                                                                                      prettier/prettier
  140:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  141:25  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  142:1   error  Replace `························` with `····························`                                                                                                                                                                                                              prettier/prettier
  143:21  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  144:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  146:17  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  147:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  148:1   error  Replace `····················` with `························`                                                                                                                                                                                                                      prettier/prettier
  149:17  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  151:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  152:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  159:16  error  Insert `⏎················`                                                                                                                                                                                                                                                          prettier/prettier
  160:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  161:13  error  Replace `})` with `····})⏎················`                                                                                                                                                                                                                                         prettier/prettier
  162:17  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  163:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  164:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  198:16  error  Insert `⏎················`                                                                                                                                                                                                                                                          prettier/prettier
  199:17  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  200:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  201:17  error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier
  202:1   error  Insert `····`                                                                                                                                                                                                                                                                       prettier/prettier

/Users/cameronhunter/workspace/github/cameronhunter/jest-codemods/src/transformers/chai-assert.js
  241:16  error  Insert `⏎················`                                                                                                                                                                                                                                      prettier/prettier
  242:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  243:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  244:13  error  Replace `)` with `····)⏎················`                                                                                                                                                                                                                       prettier/prettier
  245:17  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  246:1   error  Replace `····················` with `························`                                                                                                                                                                                                  prettier/prettier
  247:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  248:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  249:13  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  252:20  error  Insert `⏎····················`                                                                                                                                                                                                                                  prettier/prettier
  253:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  254:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  255:17  error  Replace `)` with `····)⏎····················`                                                                                                                                                                                                                   prettier/prettier
  256:1   error  Replace `····················` with `························`                                                                                                                                                                                                  prettier/prettier
  257:25  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  258:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  259:21  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  260:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  266:12  error  Insert `⏎············`                                                                                                                                                                                                                                          prettier/prettier
  267:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  268:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  269:9   error  Replace `)` with `····)⏎············`                                                                                                                                                                                                                           prettier/prettier
  270:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  271:1   error  Replace `········` with `············`                                                                                                                                                                                                                          prettier/prettier
  275:12  error  Insert `⏎············`                                                                                                                                                                                                                                          prettier/prettier
  276:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  277:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  278:1   error  Replace `········)` with `············)⏎············`                                                                                                                                                                                                           prettier/prettier
  279:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  280:17  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  281:17  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  282:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  283:21  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  284:1   error  Replace `························` with `····························`                                                                                                                                                                                          prettier/prettier
  285:29  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  286:1   error  Replace `····························` with `································`                                                                                                                                                                                  prettier/prettier
  287:29  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  288:1   error  Replace `························` with `····························`                                                                                                                                                                                          prettier/prettier
  289:21  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  290:1   error  Replace `················` with `····················`                                                                                                                                                                                                          prettier/prettier
  291:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  292:1   error  Replace `············` with `················`                                                                                                                                                                                                                  prettier/prettier
  293:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  297:8   error  Delete `.find(`                                                                                                                                                                                                                                                 prettier/prettier
  298:9   error  Replace `j.CallExpression,⏎········getAssertionExpression(chaiAssertExpression,·'fail')` with `.find(j.CallExpression,·getAssertionExpression(chaiAssertExpression,·'fail'))`                                                                                   prettier/prettier
  300:5   error  Replace `)` with `····`                                                                                                                                                                                                                                         prettier/prettier
  303:8   error  Insert `⏎········`                                                                                                                                                                                                                                              prettier/prettier
  304:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  305:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  306:5   error  Replace `)` with `····)⏎········`                                                                                                                                                                                                                               prettier/prettier
  307:1   error  Replace `········` with `············`                                                                                                                                                                                                                          prettier/prettier
  308:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  309:1   error  Replace `····` with `········`                                                                                                                                                                                                                                  prettier/prettier
  312:8   error  Insert `⏎········`                                                                                                                                                                                                                                              prettier/prettier
  313:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  314:1   error  Replace `········` with `············`                                                                                                                                                                                                                          prettier/prettier
  315:5   error  Replace `)` with `····)⏎········`                                                                                                                                                                                                                               prettier/prettier
  316:1   error  Replace `········` with `············`                                                                                                                                                                                                                          prettier/prettier
  317:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  318:1   error  Replace `····` with `········`                                                                                                                                                                                                                                  prettier/prettier
  321:8   error  Delete `.find(`                                                                                                                                                                                                                                                 prettier/prettier
  322:9   error  Replace `j.CallExpression,⏎········getAssertionExpression(chaiAssertExpression,·'property')` with `.find(j.CallExpression,·getAssertionExpression(chaiAssertExpression,·'property'))`                                                                           prettier/prettier
  324:5   error  Replace `)` with `····`                                                                                                                                                                                                                                         prettier/prettier
  325:9   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  326:1   error  Replace `············` with `················`                                                                                                                                                                                                                  prettier/prettier
  327:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  328:9   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  329:1   error  Replace `····` with `········`                                                                                                                                                                                                                                  prettier/prettier
  332:8   error  Insert `⏎········`                                                                                                                                                                                                                                              prettier/prettier
  333:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  334:1   error  Replace `········` with `············`                                                                                                                                                                                                                          prettier/prettier
  335:5   error  Replace `)` with `····)⏎········`                                                                                                                                                                                                                               prettier/prettier
  336:1   error  Replace `········` with `············`                                                                                                                                                                                                                          prettier/prettier
  337:13  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  338:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  339:9   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  340:1   error  Replace `····` with `········`                                                                                                                                                                                                                                  prettier/prettier
  343:8   error  Delete `.find(`                                                                                                                                                                                                                                                 prettier/prettier
  344:9   error  Replace `j.CallExpression,⏎········getAssertionExpression(chaiAssertExpression,·'isArray')` with `.find(j.CallExpression,·getAssertionExpression(chaiAssertExpression,·'isArray'))`                                                                             prettier/prettier
  346:5   error  Replace `)` with `····`                                                                                                                                                                                                                                         prettier/prettier
  347:9   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  348:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  349:13  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  350:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  351:17  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  352:13  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  353:1   error  Replace `············` with `················`                                                                                                                                                                                                                  prettier/prettier
  354:9   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  355:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  358:8   error  Insert `⏎········`                                                                                                                                                                                                                                              prettier/prettier
  359:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  360:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  361:5   error  Replace `)` with `····)⏎········`                                                                                                                                                                                                                               prettier/prettier
  362:1   error  Replace `········` with `············`                                                                                                                                                                                                                          prettier/prettier
  363:13  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  364:1   error  Replace `············` with `················`                                                                                                                                                                                                                  prettier/prettier
  365:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  366:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  367:13  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  368:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  369:1   error  Replace `········` with `············`                                                                                                                                                                                                                          prettier/prettier
  370:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  373:8   error  Delete `.find(`                                                                                                                                                                                                                                                 prettier/prettier
  374:9   error  Replace `j.CallExpression,⏎········getAssertionExpression(chaiAssertExpression,·'typeOf')` with `.find(j.CallExpression,·getAssertionExpression(chaiAssertExpression,·'typeOf'))`                                                                               prettier/prettier
  376:5   error  Replace `)` with `····`                                                                                                                                                                                                                                         prettier/prettier
  377:1   error  Replace `········` with `············`                                                                                                                                                                                                                          prettier/prettier
  378:13  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  379:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  380:13  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  381:1   error  Replace `········` with `············`                                                                                                                                                                                                                          prettier/prettier
  382:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  385:8   error  Delete `.find(`                                                                                                                                                                                                                                                 prettier/prettier
  386:9   error  Replace `j.CallExpression,⏎········getAssertionExpression(chaiAssertExpression,·'notTypeOf')` with `.find(j.CallExpression,·getAssertionExpression(chaiAssertExpression,·'notTypeOf'))`                                                                         prettier/prettier
  388:5   error  Replace `)` with `····`                                                                                                                                                                                                                                         prettier/prettier
  389:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  390:13  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  391:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  392:13  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  393:9   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  394:1   error  Replace `····` with `········`                                                                                                                                                                                                                                  prettier/prettier
  397:12  error  Delete `.find(`                                                                                                                                                                                                                                                 prettier/prettier
  398:13  error  Replace `j.CallExpression,⏎············getAssertionExpression(chaiAssertExpression,·assert)` with `.find(j.CallExpression,·getAssertionExpression(chaiAssertExpression,·assert))`                                                                               prettier/prettier
  400:9   error  Replace `)` with `····`                                                                                                                                                                                                                                         prettier/prettier
  401:1   error  Replace `············` with `················`                                                                                                                                                                                                                  prettier/prettier
  402:17  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  403:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  404:17  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  405:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  406:9   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  408:12  error  Insert `⏎············`                                                                                                                                                                                                                                          prettier/prettier
  409:1   error  Replace `············` with `················`                                                                                                                                                                                                                  prettier/prettier
  410:1   error  Replace `············getAssertionExpression(chaiAssertExpression,·assert.replace(/^is/,·'isNot')` with `················getAssertionExpression(⏎····················chaiAssertExpression,⏎····················assert.replace(/^is/,·'isNot')⏎················`  prettier/prettier
  411:1   error  Replace `········)` with `············)⏎············`                                                                                                                                                                                                           prettier/prettier
  412:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  413:17  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  414:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  415:17  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  416:13  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  417:1   error  Replace `········` with `············`                                                                                                                                                                                                                          prettier/prettier
  421:8   error  Delete `.find(`                                                                                                                                                                                                                                                 prettier/prettier
  422:9   error  Replace `j.CallExpression,⏎········getAssertionExpression(chaiAssertExpression,·'lengthOf')` with `.find(j.CallExpression,·getAssertionExpression(chaiAssertExpression,·'lengthOf'))`                                                                           prettier/prettier
  424:5   error  Replace `)` with `····`                                                                                                                                                                                                                                         prettier/prettier
  425:9   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  426:1   error  Replace `············` with `················`                                                                                                                                                                                                                  prettier/prettier
  427:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  428:13  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  429:9   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  430:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  436:12  error  Delete `.find(`                                                                                                                                                                                                                                                 prettier/prettier
  437:13  error  Replace `j.CallExpression,⏎············getAssertionExpression(chaiAssertExpression,·check)` with `.find(j.CallExpression,·getAssertionExpression(chaiAssertExpression,·check))`                                                                                 prettier/prettier
  439:1   error  Replace `········)` with `············`                                                                                                                                                                                                                         prettier/prettier
  440:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  441:1   error  Replace `················` with `····················`                                                                                                                                                                                                          prettier/prettier
  442:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  443:21  error  Replace `j.memberExpression(j.identifier('Object'),·j.identifier(expectation)` with `····j.memberExpression(⏎····························j.identifier('Object'),⏎····························j.identifier(expectation)⏎························`                prettier/prettier
  444:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  445:17  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  446:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  447:13  error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  448:1   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  452:8   error  Insert `⏎········`                                                                                                                                                                                                                                              prettier/prettier
  453:9   error  Insert `····`                                                                                                                                                                                                                                                   prettier/prettier
  454:5   error  Replace `})` with `····})⏎········`                                                                                                                                                                                                                             prettier/prettier

/Users/cameronhunter/workspace/github/cameronhunter/jest-codemods/src/transformers/chai-should.js
  607:9   error  Insert `⏎········`  prettier/prettier
  608:1   error  Insert `····`       prettier/prettier
  609:13  error  Insert `····`       prettier/prettier
  610:9   error  Insert `····`       prettier/prettier
  611:1   error  Insert `····`       prettier/prettier

/Users/cameronhunter/workspace/github/cameronhunter/jest-codemods/src/transformers/expect-js.js
   75:8   error  Insert `⏎········`                                                      prettier/prettier
   76:9   error  Insert `····`                                                           prettier/prettier
   77:1   error  Replace `············` with `················`                          prettier/prettier
   78:13  error  Insert `····`                                                           prettier/prettier
   79:1   error  Replace `················` with `····················`                  prettier/prettier
   80:17  error  Insert `····`                                                           prettier/prettier
   81:1   error  Replace `····················` with `························`          prettier/prettier
   82:1   error  Insert `····`                                                           prettier/prettier
   83:1   error  Insert `····`                                                           prettier/prettier
   84:21  error  Insert `····`                                                           prettier/prettier
   85:1   error  Insert `····················},⏎`                                        prettier/prettier
   87:10  error  Replace `,` with `)`                                                    prettier/prettier
   88:1   error  Replace `····})` with `········`                                        prettier/prettier
   89:1   error  Replace `········` with `············`                                  prettier/prettier
   90:1   error  Insert `····`                                                           prettier/prettier
   91:1   error  Replace `········` with `············`                                  prettier/prettier
   92:13  error  Insert `····`                                                           prettier/prettier
   93:1   error  Replace `········` with `············`                                  prettier/prettier
   95:1   error  Insert `····`                                                           prettier/prettier
   96:1   error  Replace `········` with `············`                                  prettier/prettier
   97:1   error  Insert `····`                                                           prettier/prettier
   98:1   error  Replace `········` with `············`                                  prettier/prettier
   99:13  error  Insert `····`                                                           prettier/prettier
  100:1   error  Replace `················` with `····················`                  prettier/prettier
  101:13  error  Insert `····`                                                           prettier/prettier
  102:1   error  Replace `················` with `····················`                  prettier/prettier
  103:21  error  Insert `····`                                                           prettier/prettier
  104:1   error  Replace `························` with `····························`  prettier/prettier
  105:1   error  Insert `····`                                                           prettier/prettier
  106:21  error  Insert `····`                                                           prettier/prettier
  107:1   error  Insert `····`                                                           prettier/prettier
  108:1   error  Replace `············` with `················`                          prettier/prettier
  109:1   error  Insert `····`                                                           prettier/prettier
  110:1   error  Insert `····`                                                           prettier/prettier
  111:13  error  Insert `····`                                                           prettier/prettier
  112:1   error  Replace `················` with `····················`                  prettier/prettier
  113:21  error  Insert `····`                                                           prettier/prettier
  114:1   error  Replace `····················` with `························`          prettier/prettier
  115:21  error  Insert `····`                                                           prettier/prettier
  116:1   error  Replace `····················` with `························`          prettier/prettier
  117:17  error  Insert `····`                                                           prettier/prettier
  118:1   error  Replace `············` with `················`                          prettier/prettier
  119:13  error  Insert `····`                                                           prettier/prettier
  120:1   error  Replace `············` with `················`                          prettier/prettier
  121:1   error  Insert `····`                                                           prettier/prettier
  122:1   error  Replace `············` with `················`                          prettier/prettier
  123:1   error  Insert `····`                                                           prettier/prettier
  124:1   error  Replace `····················` with `························`          prettier/prettier
  125:21  error  Insert `····`                                                           prettier/prettier
  126:1   error  Replace `····················` with `························`          prettier/prettier
  127:1   error  Insert `····`                                                           prettier/prettier
  128:1   error  Replace `················` with `····················`                  prettier/prettier
  129:1   error  Insert `····`                                                           prettier/prettier
  130:1   error  Insert `····`                                                           prettier/prettier
  131:17  error  Insert `····`                                                           prettier/prettier
  132:1   error  Replace `············` with `················`                          prettier/prettier
  133:1   error  Insert `····`                                                           prettier/prettier
  134:9   error  Insert `····`                                                           prettier/prettier
  135:1   error  Insert `····`                                                           prettier/prettier

/Users/cameronhunter/workspace/github/cameronhunter/jest-codemods/src/transformers/expect.js
  138:101  error  Insert `matcherNode`                                                                                              prettier/prettier
  139:17   error  Replace `matcherNode.arguments.length⏎············` with `.arguments.length`                                      prettier/prettier
  221:12   error  Insert `⏎············`                                                                                            prettier/prettier
  222:1    error  Replace `············` with `················`                                                                    prettier/prettier
  223:1    error  Insert `····`                                                                                                     prettier/prettier
  224:1    error  Insert `····`                                                                                                     prettier/prettier
  225:1    error  Insert `····`                                                                                                     prettier/prettier
  226:9    error  Replace `})` with `····})⏎············`                                                                           prettier/prettier
  227:1    error  Insert `····`                                                                                                     prettier/prettier
  228:17   error  Insert `····`                                                                                                     prettier/prettier
  229:1    error  Replace `················` with `····················`                                                            prettier/prettier
  230:13   error  Insert `····`                                                                                                     prettier/prettier
  231:1    error  Insert `····`                                                                                                     prettier/prettier
  234:12   error  Insert `⏎············`                                                                                            prettier/prettier
  235:13   error  Insert `····`                                                                                                     prettier/prettier
  236:1    error  Replace `················` with `····················`                                                            prettier/prettier
  237:1    error  Insert `····`                                                                                                     prettier/prettier
  238:13   error  Insert `····`                                                                                                     prettier/prettier
  239:1    error  Insert `····`                                                                                                     prettier/prettier
  240:9    error  Replace `})` with `····})⏎············`                                                                           prettier/prettier
  241:1    error  Insert `····`                                                                                                     prettier/prettier
  242:13   error  Insert `····`                                                                                                     prettier/prettier
  243:1    error  Replace `················` with `····················`                                                            prettier/prettier
  244:13   error  Insert `····`                                                                                                     prettier/prettier
  246:1    error  Replace `············` with `················`                                                                    prettier/prettier
  247:17   error  Insert `····`                                                                                                     prettier/prettier
  248:1    error  Replace `····················` with `························`                                                    prettier/prettier
  249:21   error  Insert `····`                                                                                                     prettier/prettier
  250:17   error  Insert `····`                                                                                                     prettier/prettier
  251:1    error  Replace `············` with `················`                                                                    prettier/prettier
  252:1    error  Insert `····`                                                                                                     prettier/prettier
  255:12   error  Insert `⏎············`                                                                                            prettier/prettier
  256:1    error  Replace `············` with `················`                                                                    prettier/prettier
  257:1    error  Insert `····`                                                                                                     prettier/prettier
  258:1    error  Replace `················` with `····················`                                                            prettier/prettier
  259:13   error  Insert `····`                                                                                                     prettier/prettier
  260:1    error  Insert `····`                                                                                                     prettier/prettier
  261:1    error  Replace `········})` with `············})⏎············`                                                           prettier/prettier
  262:1    error  Replace `············` with `················`                                                                    prettier/prettier
  263:1    error  Insert `····`                                                                                                     prettier/prettier
  264:17   error  Insert `····`                                                                                                     prettier/prettier
  265:1    error  Replace `············` with `················`                                                                    prettier/prettier
  266:1    error  Insert `····`                                                                                                     prettier/prettier
  267:14   error  Replace `·······`"${⏎························path.value.property.name` with `···········`"${path.value.property`  prettier/prettier
  269:21   error  Insert `········.name`                                                                                            prettier/prettier
  270:21   error  Insert `····`                                                                                                     prettier/prettier
  271:1    error  Insert `····`                                                                                                     prettier/prettier
  272:13   error  Insert `····`                                                                                                     prettier/prettier
  273:1    error  Insert `····`                                                                                                     prettier/prettier
  371:12   error  Insert `⏎············`                                                                                            prettier/prettier
  372:13   error  Insert `····`                                                                                                     prettier/prettier
  373:1    error  Replace `················` with `····················`                                                            prettier/prettier
  374:1    error  Replace `················` with `····················`                                                            prettier/prettier
  375:13   error  Insert `····`                                                                                                     prettier/prettier
  376:1    error  Replace `············` with `················`                                                                    prettier/prettier
  377:1    error  Replace `················` with `····················`                                                            prettier/prettier
  378:1    error  Insert `····`                                                                                                     prettier/prettier
  379:13   error  Insert `····`                                                                                                     prettier/prettier
  380:1    error  Replace `········})` with `············})⏎············`                                                           prettier/prettier
  381:1    error  Insert `····`                                                                                                     prettier/prettier
  382:13   error  Insert `····`                                                                                                     prettier/prettier
  383:1    error  Replace `················` with `····················`                                                            prettier/prettier
  384:1    error  Insert `····`                                                                                                     prettier/prettier
  386:13   error  Insert `····`                                                                                                     prettier/prettier
  388:1    error  Replace `············` with `················`                                                                    prettier/prettier
  389:1    error  Insert `····`                                                                                                     prettier/prettier
  392:12   error  Insert `⏎············`                                                                                            prettier/prettier
  393:13   error  Insert `····`                                                                                                     prettier/prettier
  394:1    error  Replace `················` with `····················`                                                            prettier/prettier
  395:1    error  Insert `····`                                                                                                     prettier/prettier
  396:13   error  Insert `····`                                                                                                     prettier/prettier
  397:1    error  Replace `········})` with `············})⏎············`                                                           prettier/prettier
  398:1    error  Insert `····`                                                                                                     prettier/prettier
  399:1    error  Insert `····`                                                                                                     prettier/prettier
  400:13   error  Insert `····`                                                                                                     prettier/prettier
  401:17   error  Insert `····`                                                                                                     prettier/prettier
  402:1    error  Insert `····`                                                                                                     prettier/prettier
  403:13   error  Insert `····`                                                                                                     prettier/prettier
  405:1    error  Insert `····`                                                                                                     prettier/prettier
  406:1    error  Insert `····`                                                                                                     prettier/prettier
  407:17   error  Insert `····`                                                                                                     prettier/prettier
  408:13   error  Insert `····`                                                                                                     prettier/prettier
  410:1    error  Insert `····`                                                                                                     prettier/prettier
  411:17   error  Insert `····`                                                                                                     prettier/prettier
  412:13   error  Insert `····`                                                                                                     prettier/prettier
  413:1    error  Insert `····`                                                                                                     prettier/prettier

/Users/cameronhunter/workspace/github/cameronhunter/jest-codemods/src/transformers/mocha.js
  47:12  error  Insert `⏎············`                                                                                                                                                                                                                                                                  prettier/prettier
  48:13  error  Insert `····`                                                                                                                                                                                                                                                                           prettier/prettier
  49:1   error  Insert `····`                                                                                                                                                                                                                                                                           prettier/prettier
  50:1   error  Insert `····`                                                                                                                                                                                                                                                                           prettier/prettier
  61:16  error  Insert `⏎················`                                                                                                                                                                                                                                                              prettier/prettier
  62:17  error  Insert `····`                                                                                                                                                                                                                                                                           prettier/prettier
  63:1   error  Replace `················` with `····················`                                                                                                                                                                                                                                  prettier/prettier
  64:1   error  Insert `····`                                                                                                                                                                                                                                                                           prettier/prettier
  65:1   error  Insert `····`                                                                                                                                                                                                                                                                           prettier/prettier
  66:1   error  Replace `····················` with `························`                                                                                                                                                                                                                          prettier/prettier
  67:17  error  Insert `····`                                                                                                                                                                                                                                                                           prettier/prettier
  68:13  error  Replace `})` with `····})⏎················`                                                                                                                                                                                                                                             prettier/prettier
  69:17  error  Insert `····`                                                                                                                                                                                                                                                                           prettier/prettier
  70:1   error  Replace `····················j.memberExpression(j.identifier(jestMethod),·j.identifier(modifier)` with `························j.memberExpression(⏎····························j.identifier(jestMethod),⏎····························j.identifier(modifier)⏎························`  prettier/prettier
  71:1   error  Insert `····`                                                                                                                                                                                                                                                                           prettier/prettier
  72:17  error  Insert `····`                                                                                                                                                                                                                                                                           prettier/prettier
  73:1   error  Insert `····`                                                                                                                                                                                                                                                                           prettier/prettier

/Users/cameronhunter/workspace/github/cameronhunter/jest-codemods/src/transformers/should.js
  38:13  error  Insert `⏎············`                                   prettier/prettier
  39:13  error  Insert `····`                                            prettier/prettier
  40:1   error  Insert `····`                                            prettier/prettier
  41:13  error  Insert `····`                                            prettier/prettier
  42:1   error  Replace `········})` with `············})⏎············`  prettier/prettier
  44:13  error  Insert `⏎············`                                   prettier/prettier
  45:13  error  Insert `····`                                            prettier/prettier
  46:1   error  Insert `····`                                            prettier/prettier
  47:17  error  Insert `····`                                            prettier/prettier
  48:13  error  Insert `····`                                            prettier/prettier
  49:1   error  Replace `········})` with `············})⏎············`  prettier/prettier
  53:13  error  Insert `⏎············`                                   prettier/prettier
  54:1   error  Insert `····`                                            prettier/prettier
  55:17  error  Insert `····`                                            prettier/prettier
  56:13  error  Insert `····`                                            prettier/prettier
  57:1   error  Replace `········})` with `············})⏎············`  prettier/prettier
  58:1   error  Insert `····`                                            prettier/prettier
  59:1   error  Replace `········` with `············`                   prettier/prettier
  63:13  error  Insert `⏎············`                                   prettier/prettier
  64:13  error  Insert `····`                                            prettier/prettier
  65:1   error  Insert `····`                                            prettier/prettier
  66:13  error  Insert `····`                                            prettier/prettier
  67:1   error  Insert `····`                                            prettier/prettier

/Users/cameronhunter/workspace/github/cameronhunter/jest-codemods/src/transformers/tape.js
  103:16  error  Insert `⏎················`                                                                                                                                                                                                                              prettier/prettier
  104:17  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  105:1   error  Replace `····················` with `························`                                                                                                                                                                                          prettier/prettier
  106:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  107:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  108:1   error  Replace `············})` with `················})⏎················`                                                                                                                                                                                     prettier/prettier
  109:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  110:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  111:21  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  112:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  113:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  114:21  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  115:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  116:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  117:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  118:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  120:16  error  Insert `⏎················`                                                                                                                                                                                                                              prettier/prettier
  121:17  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  122:1   error  Replace `····················` with `························`                                                                                                                                                                                          prettier/prettier
  123:21  error  Replace `property:·({·name·})·=>` with `····property:·({·name·})·=>⏎···························`                                                                                                                                                        prettier/prettier
  124:17  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  125:13  error  Replace `})` with `····})⏎················`                                                                                                                                                                                                             prettier/prettier
  126:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  127:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  128:1   error  Replace `············` with `················`                                                                                                                                                                                                          prettier/prettier
  132:16  error  Insert `⏎················`                                                                                                                                                                                                                              prettier/prettier
  133:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  134:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  135:1   error  Replace `····················` with `························`                                                                                                                                                                                          prettier/prettier
  136:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  137:1   error  Replace `························` with `····························`                                                                                                                                                                                  prettier/prettier
  138:17  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  139:13  error  Replace `})` with `····})⏎················`                                                                                                                                                                                                             prettier/prettier
  140:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  141:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  142:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  144:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  145:21  error  Replace `logWarning(`"t.${oldPropertyName}"·is·currently·not·supported`,·p` with `····logWarning(⏎····························`"t.${oldPropertyName}"·is·currently·not·supported`,⏎····························p⏎························`              prettier/prettier
  146:1   error  Replace `····················` with `························`                                                                                                                                                                                          prettier/prettier
  147:17  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  149:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  150:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  151:1   error  Replace `····················` with `························`                                                                                                                                                                                          prettier/prettier
  152:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  153:21  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  154:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  155:25  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  156:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  157:21  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  158:1   error  Replace `····················` with `························`                                                                                                                                                                                          prettier/prettier
  159:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  160:1   error  Replace `····························` with `································`                                                                                                                                                                          prettier/prettier
  161:1   error  Replace `································oldPropertyName·===·'throws'·?·'toThrow'` with `····································oldPropertyName·===·'throws'⏎········································?·'toThrow'⏎·······································`  prettier/prettier
  162:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  163:29  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  164:1   error  Replace `························` with `····························`                                                                                                                                                                                  prettier/prettier
  165:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  166:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  167:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  168:33  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  169:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  170:37  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  171:1   error  Replace `····························` with `································`                                                                                                                                                                          prettier/prettier
  172:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  173:25  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  174:21  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  175:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  176:21  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  177:1   error  Replace `························` with `····························`                                                                                                                                                                                  prettier/prettier
  178:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  179:21  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  180:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  181:17  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  182:1   error  Replace `····················` with `························`                                                                                                                                                                                          prettier/prettier
  183:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  184:1   error  Replace `····················` with `························`                                                                                                                                                                                          prettier/prettier
  185:21  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  186:1   error  Replace `························` with `····························`                                                                                                                                                                                  prettier/prettier
  187:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  188:21  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  189:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  191:17  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  192:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  193:21  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  194:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  196:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  197:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  201:16  error  Insert `⏎················`                                                                                                                                                                                                                              prettier/prettier
  202:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  203:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  204:1   error  Replace `····················` with `························`                                                                                                                                                                                          prettier/prettier
  205:17  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  206:1   error  Replace `············})` with `················})⏎················`                                                                                                                                                                                     prettier/prettier
  207:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  208:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  212:16  error  Insert `⏎················`                                                                                                                                                                                                                              prettier/prettier
  213:17  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  214:13  error  Replace `})` with `····})⏎················`                                                                                                                                                                                                             prettier/prettier
  215:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  216:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  217:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  218:1   error  Replace `························` with `····························`                                                                                                                                                                                  prettier/prettier
  219:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  220:29  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  221:29  error  Replace `if·(tapeOptionKey·===·'skip'·&&·tapeOptionValue·===·true` with `····if·(⏎····································tapeOptionKey·===·'skip'·&&⏎····································tapeOptionValue·===·true⏎································`        prettier/prettier
  222:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  223:29  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  225:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  226:33  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  227:1   error  Replace `····································` with `········································`                                                                                                                                                          prettier/prettier
  228:37  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  229:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  230:29  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  231:1   error  Replace `························` with `····························`                                                                                                                                                                                  prettier/prettier
  233:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  234:29  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  235:1   error  Replace `························` with `····························`                                                                                                                                                                                  prettier/prettier
  236:21  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  237:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  239:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  240:1   error  Replace `····················` with `························`                                                                                                                                                                                          prettier/prettier
  241:17  error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier
  243:1   error  Replace `················` with `····················`                                                                                                                                                                                                  prettier/prettier
  244:1   error  Insert `····`                                                                                                                                                                                                                                           prettier/prettier

/Users/cameronhunter/workspace/github/cameronhunter/jest-codemods/src/utils/finale.js
  23:8   error  Insert `⏎········`                              prettier/prettier
  24:1   error  Replace `········` with `············`          prettier/prettier
  25:1   error  Insert `····`                                   prettier/prettier
  26:1   error  Replace `············` with `················`  prettier/prettier
  27:1   error  Insert `····`                                   prettier/prettier
  28:9   error  Insert `····`                                   prettier/prettier
  29:1   error  Replace `····})` with `········})⏎········`     prettier/prettier
  30:1   error  Insert `····`                                   prettier/prettier
  31:1   error  Replace `········` with `············`          prettier/prettier
  32:1   error  Insert `····`                                   prettier/prettier
  33:13  error  Insert `····`                                   prettier/prettier
  34:1   error  Replace `············` with `················`  prettier/prettier
  35:9   error  Insert `····`                                   prettier/prettier
  36:1   error  Replace `············` with `················`  prettier/prettier
  37:9   error  Insert `····`                                   prettier/prettier
  38:1   error  Insert `····`                                   prettier/prettier

/Users/cameronhunter/workspace/github/cameronhunter/jest-codemods/src/utils/imports.js
   20:8   error  Insert `⏎········`                                                                prettier/prettier
  213:16  error  Insert `⏎················`                                                        prettier/prettier
  214:1   error  Insert `····`                                                                     prettier/prettier
  215:17  error  Insert `····`                                                                     prettier/prettier
  216:1   error  Replace `····················` with `························`                    prettier/prettier
  217:1   error  Insert `····`                                                                     prettier/prettier
  218:1   error  Insert `····`                                                                     prettier/prettier
  219:1   error  Replace `············})` with `················})⏎················`               prettier/prettier
  220:1   error  Insert `····`                                                                     prettier/prettier
  221:21  error  Replace `prop·=>` with `····prop·=>⏎···························`                  prettier/prettier
  222:1   error  Replace `················` with `····················`                            prettier/prettier
  224:1   error  Insert `····`                                                                     prettier/prettier
  225:1   error  Insert `····`                                                                     prettier/prettier
  226:1   error  Replace `····················` with `························`                    prettier/prettier
  227:1   error  Insert `····`                                                                     prettier/prettier
  228:25  error  Insert `····`                                                                     prettier/prettier
  229:1   error  Insert `····`                                                                     prettier/prettier
  230:29  error  Insert `····`                                                                     prettier/prettier
  231:25  error  Insert `····`                                                                     prettier/prettier
  232:1   error  Insert `····`                                                                     prettier/prettier
  233:1   error  Replace `························` with `····························`            prettier/prettier
  234:1   error  Insert `····`                                                                     prettier/prettier
  235:17  error  Insert `····`                                                                     prettier/prettier
  236:1   error  Insert `····`                                                                     prettier/prettier
  240:16  error  Insert `⏎················`                                                        prettier/prettier
  241:1   error  Replace `················` with `····················`                            prettier/prettier
  242:1   error  Insert `····`                                                                     prettier/prettier
  243:1   error  Replace `····················` with `························`                    prettier/prettier
  244:1   error  Insert `····`                                                                     prettier/prettier
  245:17  error  Replace `property:·node·=>` with `····property:·node·=>⏎·······················`  prettier/prettier
  246:1   error  Insert `····`                                                                     prettier/prettier

/Users/cameronhunter/workspace/github/cameronhunter/jest-codemods/src/utils/proxyquire.js
   34:12  error  Insert `⏎············`                                                                 prettier/prettier
   35:1   error  Insert `····`                                                                          prettier/prettier
   36:9   error  Replace `})` with `····})⏎············`                                                prettier/prettier
   37:1   error  Insert `····`                                                                          prettier/prettier
   38:13  error  Insert `····`                                                                          prettier/prettier
   39:1   error  Replace `················` with `····················`                                 prettier/prettier
   40:13  error  Insert `····`                                                                          prettier/prettier
   42:1   error  Insert `····`                                                                          prettier/prettier
   43:13  error  Insert `····`                                                                          prettier/prettier
   44:1   error  Insert `····`                                                                          prettier/prettier
   45:1   error  Replace `················` with `····················`                                 prettier/prettier
   46:17  error  Insert `····`                                                                          prettier/prettier
   47:13  error  Insert `····`                                                                          prettier/prettier
   49:1   error  Replace `············` with `················`                                         prettier/prettier
   50:13  error  Insert `····`                                                                          prettier/prettier
   52:1   error  Replace `············` with `················`                                         prettier/prettier
   53:1   error  Insert `····`                                                                          prettier/prettier
   54:17  error  Insert `····`                                                                          prettier/prettier
   55:1   error  Replace `············` with `················`                                         prettier/prettier
   56:1   error  Insert `····`                                                                          prettier/prettier
   58:1   error  Insert `····`                                                                          prettier/prettier
   59:1   error  Replace `················` with `····················`                                 prettier/prettier
   60:1   error  Insert `····`                                                                          prettier/prettier
   61:1   error  Insert `····`                                                                          prettier/prettier
   62:25  error  Insert `····`                                                                          prettier/prettier
   63:1   error  Insert `····`                                                                          prettier/prettier
   64:1   error  Replace `····················` with `························`                         prettier/prettier
   65:1   error  Insert `····`                                                                          prettier/prettier
   66:25  error  Insert `····`                                                                          prettier/prettier
   67:1   error  Insert `····`                                                                          prettier/prettier
   68:17  error  Insert `····`                                                                          prettier/prettier
   69:1   error  Replace `············` with `················`                                         prettier/prettier
   70:1   error  Insert `····`                                                                          prettier/prettier
   71:17  error  Insert `····`                                                                          prettier/prettier
   72:1   error  Replace `················ast` with `····················ast⏎························`  prettier/prettier
   73:1   error  Replace `····················` with `····························`                     prettier/prettier
   74:17  error  Insert `········`                                                                      prettier/prettier
   75:1   error  Insert `····`                                                                          prettier/prettier
   76:21  error  Insert `····`                                                                          prettier/prettier
   77:1   error  Replace `························` with `····························`                 prettier/prettier
   78:1   error  Insert `····`                                                                          prettier/prettier
   80:17  error  Insert `····`                                                                          prettier/prettier
   81:1   error  Replace `····················` with `························`                         prettier/prettier
   82:1   error  Insert `····`                                                                          prettier/prettier
   83:1   error  Replace `························` with `····························`                 prettier/prettier
   84:25  error  Insert `····`                                                                          prettier/prettier
   85:1   error  Insert `····`                                                                          prettier/prettier
   86:1   error  Replace `····················` with `························`                         prettier/prettier
   87:17  error  Insert `····`                                                                          prettier/prettier
   89:1   error  Insert `····`                                                                          prettier/prettier
   90:21  error  Insert `····`                                                                          prettier/prettier
   91:1   error  Insert `····`                                                                          prettier/prettier
   92:1   error  Replace `························` with `····························`                 prettier/prettier
   93:1   error  Insert `····`                                                                          prettier/prettier
   94:25  error  Insert `····`                                                                          prettier/prettier
   95:1   error  Replace `····························` with `································`         prettier/prettier
   96:29  error  Insert `····`                                                                          prettier/prettier
   97:1   error  Insert `····`                                                                          prettier/prettier
   98:21  error  Insert `····`                                                                          prettier/prettier
   99:1   error  Insert `····`                                                                          prettier/prettier
  100:1   error  Replace `························` with `····························`                 prettier/prettier
  101:1   error  Insert `····`                                                                          prettier/prettier
  102:17  error  Insert `····`                                                                          prettier/prettier
  103:1   error  Replace `············` with `················`                                         prettier/prettier
  104:17  error  Insert `····`                                                                          prettier/prettier
  105:1   error  Replace `············` with `················`                                         prettier/prettier
  107:1   error  Insert `····`                                                                          prettier/prettier
  108:17  error  Insert `····`                                                                          prettier/prettier
  109:1   error  Replace `············` with `················`                                         prettier/prettier
  110:1   error  Insert `····`                                                                          prettier/prettier
  111:1   error  Insert `····`                                                                          prettier/prettier

/Users/cameronhunter/workspace/github/cameronhunter/jest-codemods/src/utils/quote-style.js
  11:8   error  Insert `⏎········`                      prettier/prettier
  12:1   error  Insert `····`                           prettier/prettier
  13:1   error  Insert `····`                           prettier/prettier
  14:5   error  Replace `})` with `····})⏎········`     prettier/prettier
  15:1   error  Replace `········` with `············`  prettier/prettier
  16:9   error  Insert `····`                           prettier/prettier
  17:1   error  Insert `····`                           prettier/prettier
  18:13  error  Insert `····`                           prettier/prettier
  19:1   error  Insert `····`                           prettier/prettier
  20:9   error  Insert `····`                           prettier/prettier
  21:1   error  Insert `····`                           prettier/prettier
  22:9   error  Insert `····`                           prettier/prettier
  23:1   error  Insert `····`                           prettier/prettier

/Users/cameronhunter/workspace/github/cameronhunter/jest-codemods/src/utils/tape-ava-helpers.js
  134:8   error  Insert `⏎········`                                                                                                                                                                                                                                          prettier/prettier
  135:1   error  Replace `········` with `············`                                                                                                                                                                                                                      prettier/prettier
  136:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  137:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  138:5   error  Replace `})` with `····})⏎········`                                                                                                                                                                                                                         prettier/prettier
  139:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  140:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  141:9   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  142:9   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  143:1   error  Replace `············` with `················`                                                                                                                                                                                                              prettier/prettier
  144:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  145:13  error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  147:1   error  Replace `············` with `················`                                                                                                                                                                                                              prettier/prettier
  148:17  error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  149:1   error  Replace `····················` with `························`                                                                                                                                                                                              prettier/prettier
  150:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  151:21  error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  152:1   error  Replace `····················` with `························`                                                                                                                                                                                              prettier/prettier
  153:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  154:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  155:33  error  Replace `j.memberExpression(j.identifier('t'),·j.identifier(key)` with `····j.memberExpression(⏎········································j.identifier('t'),⏎········································j.identifier(key)⏎····································`  prettier/prettier
  156:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  157:29  error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  158:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  159:21  error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  160:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  161:9   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  162:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  171:8   error  Insert `⏎········`                                                                                                                                                                                                                                          prettier/prettier
  172:9   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  173:1   error  Replace `············` with `················`                                                                                                                                                                                                              prettier/prettier
  174:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  175:1   error  Replace `····})` with `········})⏎········`                                                                                                                                                                                                                 prettier/prettier
  176:1   error  Replace `········` with `············`                                                                                                                                                                                                                      prettier/prettier
  177:9   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  178:1   error  Replace `············` with `················`                                                                                                                                                                                                              prettier/prettier
  180:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  181:13  error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  182:1   error  Replace `················` with `····················`                                                                                                                                                                                                      prettier/prettier
  183:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  184:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  185:21  error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  186:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier
  187:1   error  Replace `············` with `················`                                                                                                                                                                                                              prettier/prettier
  188:1   error  Replace `········` with `············`                                                                                                                                                                                                                      prettier/prettier
  189:1   error  Insert `····`                                                                                                                                                                                                                                               prettier/prettier

✖ 735 problems (735 errors, 0 warnings)
  735 errors, 0 warnings potentially fixable with the `--fix` option.

info If you think this is a bug, please open a bug report with the information provided in "/Ucameronhunter/chunter/workspace/github/cameronhunter/jest-codemods/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Reply:
Hm. I cannot reproduce this. CI also runs these checks on every commit... Must be something related to the setup.